### PR TITLE
Smaller changes in openrct2.d.ts to better reflect the actual API implementation.

### PR DESCRIPTION
**Pull request by Sadret**

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@
 - Improved: [#10150] Ride stations are now properly checked if they’re sheltered.
 - Improved: [#10664, #16072] Visibility status can be modified directly in the Tile Inspector's list.
 - Improved: [#16251] Plugin API handles null values better.
-- Improved: [#16251] openrct2.d.ts: mark 'callback' argument of queryAction and executeAction as optional.
+- Improved: [#16251] Smaller changes in openrct2.d.ts to better reflect the actual API implementation.
```
Comment (duncanspumpkin) [inline distribution/changelog.txt:20]: This description is void of useful information. At least the old description meant something

Comment (Sadret) [reply distribution/changelog.txt:20]: Yes, I saw the problem but wasnt sure if I should add 4 lines to the changelog for 4 lines of changed code.

Comment (Sadret) [reply distribution/changelog.txt]: Better now?

**Review by IntelOrca — APPROVED** [review]

File: distribution/changelog.txt (hunk 1 above)
Comment (duncanspumpkin) [reply distribution/changelog.txt]: Yup that's fine

**Pull request merged**